modules/ROOT/pages/add-tags.adoc

```diff
@@ -1,14 +1,18 @@
 = Identifying APIs to Govern
 
-APIs that meet the filter criteria in API Governance profiles become _governed APIs_. Identifying the APIs to govern before creating your governance profiles helps ensure that you are governing the APIs in the way you want to govern them and that conformance notifications are sent only to the API owners who are ready to begin making their APIs conformant to your governance profiles.
+APIs that meet the filter criteria in API Governance profiles become _governed APIs_. 
+Identify the APIs to govern before creating your governance profiles to help ensure that:
+
+* You are governing only the APIs that you want to govern
```

Comment:
```suggestion
* You are governing only the APIs that you want to govern.
```

Reply:
Is this necessary to say? Or is there an alternative way to say it?

Reply:
It's important to say this, because they set this in Exchange and based on filters in API Governance profiles, the validation is run against filtered APIs. This is crucial for customers to understand. If that isn't clear, what would have made that clear to you?

Reply:
If they don't filter them correctly, a) they might be paying a lot more for API Gov than they want and b) they might be annoying lots of developers with notifications and generating a lot of angst among their staff, creating more work for themselves, etc. Do you think I need to be more explicit in explaining that?

Reply:
The way it's worded now makes it seem like it's stating the obvious. I guess something like "The list of APIs to govern is correct." 

I don't know. Maybe my issue is with the Identify intro. Or maybe there's no issue at all. Your call.

Reply:
Thanks for the explanation. 
This is one of those things that's much more obvious in a video/demo. I'll be adding a graphic which will also make it more clear. I think I'll leave it as it is for now.